### DISCUSSION TITLE
Why can't select text in the login window?

### DISCUSSION THREAD

**Discussion by HubKing** [neutrinolabs/xrdp#2331] [Q&A]
When the cursor is at the end of a text field on the Login dialogue, pressing shift+home does not select the text. Nor does dragging with the mouse. Is this a bug or an intentional behaviour?

If you want to know why I would do that, I often use that to clear a text field (e.g., to enter a different user name).

**Comment by matt335672** [comment]
At the moment, a whole-field-select doesn't seem to be implemented, either with the mouse or with the keyboard.
